src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -382,6 +383,7 @@ namespace AirLoopHVACDOAS {
 
                 thisSplitter.name = UtilityRoutines::makeUPPER(thisObjectName);
                 thisSplitter.InletNodeName = UtilityRoutines::makeUPPER(fields.at("inlet_node_name").get<std::string>());
+                thisSplitter.InletNodeNum = UtilityRoutines::FindItemInList(thisSplitter.InletNodeName, state.dataLoopNodes->NodeID);
```

Comment:
Why use `FindItemInList` instead of `GetOnlySingleNode` (which will register this node connection in the BND output)?

Reply:
The GetOnlySingleNode requires an enum of DataLoopNode::ConnectionObjectType. The current object of AirLoopHVAC:Splitter is not listed in the connection type. I am going to post a new issue to add more objects in the connection type. I will make changes after new objects are listed in the connection type. For the time being, I keep as is.